Makefile

```diff
@@ -18,6 +18,9 @@ tests: test
 api-tests:
 	SNUBA_SETTINGS=test pytest -vv tests/*_api.py
 
+backend-typing:
+	mypy snuba --strict --warn-unreachable --config-file mypy.ini --ignore-missing-imports
```

Comment:
I don't think you should add the `ignore-missing-imports`. We already do it package by package in mypy.ini.
So `ignore-missing-imports` is by default False.